deploy/coins/ethereum/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Prepare Environment
+FROM alpine:latest as builder
+
+ENV GETH_VERSION=1.10.23
+
+WORKDIR /home/ethereum
+
+RUN apk update \
+	&& apk --no-cache add ca-certificates gnupg bash su-exec
+
+# Get Binary
+RUN wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-${GETH_VERSION}-d901d853.tar.gz\
+	&& wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-${GETH_VERSION}-d901d853.tar.gz.asc
+
+# Verify Binary
+# refer to https://geth.ethereum.org/downloads/#openpgp_signatures
+# for the PGP keys of builders and developers
+ENV KEYS F9585DE6 C2FF8BBF 9BA28146 7B9E2481 D2A67EAC E058A81C 05A5DDF0 1CCB7DD2
+
+RUN gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys ${KEYS} \
+	&& gpg --verify geth-linux-amd64-${GETH_VERSION}-d901d853.tar.gz.asc geth-linux-amd64-${GETH_VERSION}-d901d853.tar.gz
```

Comment:
Can we enforce its signed by the build server and at least 1 developer? Using 2/3 may be better yet I'm not sure how actively the devs sign.